### PR TITLE
ASM - Expander - Jira Notification

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-hq.paloaltonetworks.local/browse/EXPANDR-3213

## Description
Adding Jira for notification purposes.  Taking this opportunity to make some parts of the parent playbook as sub-playbooks for organization.

## Screenshots
n/a

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [X] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@ShirleyDenkberg , can you please review when possible

**Comment by johnnywilkes** [comment]
@capanw / @BigEasyJ , please review when possible.  This error will clear, when #27795 goes through:
https://app.circleci.com/pipelines/github/demisto/content/279298/workflows/836d894b-d701-4049-8b2b-75a1baed8e82/jobs/709062

@mmhw , I assume that you will be able to bypass this error? https://github.com/demisto/content/actions/runs/5489464476/jobs/10003653319?pr=27987

**Comment by mmhw** [comment]
Hi @johnnywilkes,
For me it looks good, please update when it is possible to merge it.

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Path_Rules.yml
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Path_Rules.yml:6]:
```suggestion
description: This playbook returns "RemediationAction" options based on the return from the Remediation Path Rules API, or defaults to the data collection task.
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Path_Rules.yml
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Path_Rules.yml:1391]:
```suggestion
      description: Determines the next action (remediation or ticket) based on user input. Options are based on if the ServiceNowV2 integration is setup and/or the alert meets the requirement for automated remediation.
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml:2847]:
```suggestion
        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of the command:
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml:3075]:
```suggestion
        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of the command:
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Path_Rules.yml
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Path_Rules.yml:1462]:
```suggestion
        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of the command:
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Path_Rules.yml
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Path_Rules.yml:1661]:
```suggestion
      description: Playbook that populates remediation objectives field used to display the remediation actions to the end user.
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Path_Rules_README.md
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Path_Rules_README.md:1]:
```suggestion
This playbook returns "RemediationAction" options based on the return from the Remediation Path Rules API, or defaults to the data collection task.
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Path_Rules_README.md
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Path_Rules_README.md:9]:
```suggestion
Cortex ASM - Remediation Objectives
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ServiceNow_Notification.yml
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ServiceNow_Notification.yml:259]:
```suggestion
  description: Body of the notification (email or ticket) sent to the potential service owner.
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ServiceNow_Notification_README.md
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ServiceNow_Notification_README.md:9]:
```suggestion
Cortex ASM - Remediation Guidance
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ServiceNow_Notification_README.md
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ServiceNow_Notification_README.md:13]:
```suggestion
ServiceNow v2
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ServiceNow_Notification_README.md
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ServiceNow_Notification_README.md:17]:
```suggestion
GridFieldSetup
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ServiceNow_Notification_README.md
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ServiceNow_Notification_README.md:21]:
```suggestion
servicenow-create-ticket
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ServiceNow_Notification_README.md
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ServiceNow_Notification_README.md:29]:
```suggestion
| OwnerNotificationBody | Body of the notification \(email or ticket\) sent to the potential service owner. |  | Required |
```

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_23.md
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_23.md:13]:
```suggestion
Added the "Cortex ASM - ServiceNow Notification" playbook that is used to create ServiceNow tickets directed toward service owners to notify them of their internet exposures. (Available from Cortex XSOAR 6.8.0).
```

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_23.md
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_23.md:17]:
```suggestion
Updated the playbook to incorporate a Jira check and a dynamic data collection task.
```

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_23.md
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_23.md:21]:
```suggestion
Added the "Cortex ASM - Jira Notification" playbook that is used to create Jira tickets directed toward service owners to notify them of their internet exposures. (Available from Cortex XSOAR 6.8.0).
```

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_23.md
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_23.md:25]:
```suggestion
Added the "Cortex ASM - Email Notification" playbook that is used to send email notifications to service owners to notify them of their internet exposures. (Available from Cortex XSOAR 6.8.0).
```

**Comment by ShirleyDenkberg** [comment]
@mmhw Doc review completed.

File: Packs/CortexAttackSurfaceManagement/.pack-ignore
Comment (capanw) [inline Packs/CortexAttackSurfaceManagement/.pack-ignore:7]: @johnnywilkes This was intentionally removed?

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml
Comment (capanw) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml:554]: HTML format is not needed to be specified anymore?

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Jira_Notification.yml
Comment (capanw) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Jira_Notification.yml:135]: Observed this in few other places. Is TIMESTAMP a string? I'm guessing it should get the ISO time format and in the playbook run, I saw that the value is literal TIMESTAMP instead of an actual timestamp.

**Comment by capanw** [comment]
@johnnywilkes Great work. Added few comments.

File: Packs/CortexAttackSurfaceManagement/.pack-ignore
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/.pack-ignore:7]: yes, that effectively disabled a validation error that was in Cortex_ASM_-_ASM_Alert.yml and got moved to Cortex_ASM_-_ServiceNow_Notification.yml

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml:554]: This GIT diff is really confusing and not sure why it is comparing these two tasks that are totally different.  All of the "Select Remediation Action" data collection tasks have been moved to "Cortex ASM - Remediation Path Rules" as a single task and it is using `format: html` 
https://github.com/johnnywilkes/content-jw/blob/14a3c5b116527d9cab0277e50686ab9c06c0a26f/Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Path_Rules.yml#L1417

File: Packs/CortexAttackSurfaceManagement/pack_metadata.json
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/pack_metadata.json:1]: I might look into an automation to alphabetize this file, if the `update-release-notes` command automatically changes it.

**Review by BigEasyJ — APPROVED** [review]

**Comment by BigEasyJ** [comment]
Reviewed test incidents, didn't see anything major that needs changing.

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert_README.md
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert_README.md:10]: the sdk doesn't alphabetize so I don't really see a great reason to

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Email_Notification.yml
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Email_Notification.yml:306]: yes, good catch, will change

Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Email_Notification.yml]: [update input](https://github.com/demisto/content/pull/27987/commits/b6e5e8882e9f7dc213018ac91c5495fad1c1e549)

**Comment by johnnywilkes** [comment]
@mmhw , this has been approved from our side.  Please merge when possible, please.

**Comment by johnnywilkes** [comment]
@mmhw , sorry for the back and forth, our QA found a potential issue I want to look into before we merge

**Comment by johnnywilkes** [comment]
@mmhw , so the UI fix should be in Sunday.  So after verified Monday I will take this out of draft stage

**Comment by johnnywilkes** [comment]
@mmhw , we are good to merge now, please do when possible

**Pull request merged**
